TMGToolbox/src/input_output/export_network_package.py

```diff
@@ -254,6 +255,18 @@ def _batchout_lines(self, temp_folder, zf):
             self._export_blank_batch_file(export_file, "lines")
             self.TRACKER.completeTask()
         else:
+            # check if the description is empty or has single quote
+            network = self.Scenario.get_network()
+            for line in network.transit_lines():
+                ln_description = line.description
+                if len(ln_description) == 0:
+                    line.description = "No Description"
+                elif "'" in ln_description:
+                    line.description = ln_description.replace("'","`")
+                else:
+                    pass
```

Comment:
There is no need to have an else that does nothing.  These two lines can be removed.